addons/rating/models/rating.py

```diff
@@ -53,12 +70,43 @@ def _compute_rating_image(self):
             except (IOError, OSError):
                 rating.rating_image = False
 
-    @api.multi
     @api.depends('rating')
     def _compute_rating_text(self):
-        text = {10: _('Satisfied'), 5: _('Not satisfied'), 1: _('Highly dissatisfied')}
         for rating in self:
-            rating.rating_text = text[rating.rating] or _('No rating yet')
+            if rating.rating >= 7:
```

Comment:
@jem-odoo  Why harcoded thresholds? Can't you, at least, define constants with these values?